applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -4,3 +4,399 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
I get a warning from ascs.c when I pristine build headset:
![image](https://user-images.githubusercontent.com/89020256/171000667-a1c24a51-1074-4d29-ac92-108c13efa959.png)

Reply:
I was thought new Zephyr cherry-pick solved this :(
I'm still working on figuring out the proper fix commit.

Reply:
Thank you Erik for the help on cherry-pick, this issue fixed afte rebased.